services/fischcard.service.ts

```diff
@@ -12,18 +12,14 @@ type QueryRegExType = { [key: string]: { $regex: string; $options: string } };
 export const getAllCardsByQuery = async (
   query: QueryRegExType
 ): Promise<CardPayload[]> => {
-  try {
-    //connect to db
-    db;
-    //take cards
-    const allCardsByQuery = Card.find(query).sort({
-      date: "asc",
-    });
+  //connect to db
+  db;
+  //take cards
+  const allCardsByQuery = Card.find(query).sort({
+    date: "asc",
+  });
 
-    return allCardsByQuery as Promise<CardPayload[]>;
-  } catch (error) {
-    throw error;
-  }
+  return allCardsByQuery as Promise<CardPayload[]>;
```

Comment:
**nitpick:** Such type casting is considered bad practice. When your schema changes, you will overwrite it with outdated type making it possible to get some bugs

Reply:
Im thinking the same, but i have mo idea